radiasoft/beamsim/codes/elegant.sh

```diff
@@ -1,3 +1,90 @@
 #!/bin/bash
 codes_dependencies sdds
 codes_download https://depot.radiasoft.org/foss/elegant-28.1.0-1.fedora.21.openmpi.x86_64.rpm
+
+ELEGANT_DOCS=/usr/share/doc/elegant
```

Comment:
I think we should use lower case variables for non-exported values. It's a way to distinguish that a variable is not being passed on via "env".